android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,283 @@
+                fontWeight = FontWeight.Bold,
+                color = Color.White
+            )
+
+            Spacer(
+                modifier = Modifier
+                    .height(MaterialTheme.spacing.medium)
+            )
+
+            FavoriteButton(
+                isFavourite = movieDetailsViewState.isFavorite,
+                modifier = Modifier
+                    .padding(
+                        start = MaterialTheme.spacing.medium,
+                        bottom = MaterialTheme.spacing.medium
+                    )
+                    .size(30.dp),
+                onClick = { }
+            )
+        }
+    }
+}
+
+@Composable
+fun MovieDetailsOverview(
+    modifier: Modifier = Modifier,
+    movieDetailsViewState: MovieDetailsViewState
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        Text(
+            text = stringResource(id = R.string.overview),
+            maxLines = 1,
+            modifier = Modifier
+                .fillMaxWidth(),
+            fontSize = 18.sp,
+            fontWeight = FontWeight.Bold
+        )
+
+        Spacer(
+            modifier = Modifier
+                .height(MaterialTheme.spacing.small)
+        )
+
+        Text(
+            text = movieDetailsViewState.overview,
+            maxLines = 7,
+            modifier = Modifier
+                .fillMaxWidth(),
+            fontSize = 14.sp
+        )
+    }
+}
+
+@Composable
+fun MovieDetailsCrew(
+    modifier: Modifier = Modifier,
+    movieDetailsViewState: MovieDetailsViewState
+) {
+    Column(
+        modifier = modifier
+            .heightIn(min = 50.dp, max = 400.dp)
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        LazyVerticalGrid(
+            columns = GridCells.Fixed(3),
+            verticalArrangement = Arrangement.spacedBy(MaterialTheme.spacing.medium),
+            userScrollEnabled = false
+        ) {
+            items(
+                items = movieDetailsViewState.crew,
+                key = { crewman -> crewman.id }
+            ) { crewman ->
+                CrewItem(
+                    item = CrewItemViewState(
+                        crewman.name,
+                        crewman.job
+                    )
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun MovieDetailsCast(
+    modifier: Modifier = Modifier,
+    movieDetailsViewState: MovieDetailsViewState
+) {
+    Column(
+        modifier = modifier
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        Text(
+            text = stringResource(id = R.string.top_billed_cast),
+            maxLines = 1,
+            modifier = Modifier
+                .fillMaxWidth(),
+            fontSize = 18.sp,
+            fontWeight = FontWeight.Bold
+        )
```

Comment:
Since this kind of text is used multiple times ("Top billed cast" and "Overview") it would make sense to extract it as a separate composable (again, remember DRY principle)